src/pages/home/ReportScreen.js

```diff
@@ -608,6 +626,9 @@ export default compose(
             isSidebarLoaded: {
                 key: ONYXKEYS.IS_SIDEBAR_LOADED,
             },
+            allReportActions: {
+                key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+            },
```

Comment:
yeah, I wondered about this as well and originally tried a [different method](https://github.com/Expensify/App/pull/36934/files/d436ffe6c5ba59673ced29ffc171a0241b13275b#diff-dd4bfa50713397aca8cb40145317936ab0affe19abc2a9c24d6c8849ed75dc9bR619-R627) where we loaded only the `transactionThreadReportActions`. That method had its own pitfalls though - we ran into trouble with the report actions loading correctly on the first view of the report (discussed [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1708527161578899)). 

I'll look into it again and see if there's an alternative, since loading all the report actions is less than ideal for sure.

Reply:
updated!